src/relations/mysql.py

```diff
@@ -142,6 +142,9 @@ def _on_mysql_relation_broken(self, event: RelationBrokenEvent) -> None:
         if not self.charm.unit.is_leader():
             return
 
+        if len(self.charm.model.relations[LEGACY_MYSQL]) > 1:
```

Comment:
add a comment for why you're doing this

Reply:
addressed in 2f6e6e434f2bab7f2a508153eb234fb73edf2bed

Reply:
`s/osm-mysql/mysql/` ?